manifests/config/bind.pp

```diff
@@ -25,12 +25,16 @@
   $ssl_key = undef,
 )
 {
-  qpid::config_cmd { "bind queue to exchange and filter messages that deal with ${title}":
+  $cmd = "bind queue to exchange and filter messages that deal with ${title}"
+  qpid::config_cmd { $cmd:
     command  => "bind ${exchange} ${queue} ${title}",
     unless   => "exchanges ${exchange} -r | grep ${title}",
     hostname => $hostname,
     port     => $port,
     ssl_cert => $ssl_cert,
     ssl_key  => $ssl_key,
   }
+
+  Qpid::Config::Exchange <| exchange == $exchange |> -> Qpid::Config_cmd[$cmd]
+  Qpid::Config::Queue <| queue == $queue |> -> Qpid::Config_cmd[$cmd]
```

Comment:
Correct. It also works if they aren't created/managed (at least, there's tests for that and I don't see any errors)